CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+* **BREAKING:** Remove `firebase list` command.
```

Comment:
I think present, not future-tense grammar. "Removes"

Also how about calling out replacements where appropriate (`projects:list` and `apps:sdkconfig web`)